src/test/java/org/dcsa/api_validator/ovs/v2/GetEventsTest.java

```diff
@@ -353,7 +353,7 @@ public void testEventCreatedDateTimeFormats() {
         given().
                 auth().
                 oauth2(Configuration.accessToken).
-                queryParam("eventCreatedDateTime:eq", "2021-08-09T15:11:45.281524+02:00").
+                queryParam("eventCreatedDateTime:eq", "2021-01-09T14:12:56+01").
```

Comment:
```suggestion
                queryParam("eventCreatedDateTime:eq", "2021-01-09T14:12:56+01:00").
```

Reply:
isn't this more correct? (At least it is aligned with other places...)

Reply:
I think it's implicit, but I can add it.